2025-02-26 20:40:45.279 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration family_health_tracker which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-02-26 20:40:45.280 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration family_health_tracker which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-02-26 20:40:45.329 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event logging_changed[L]>
2025-02-26 20:40:45.330 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event logging_changed[L]>
2025-02-26 20:40:45.330 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=logger, service=set_default_level>
2025-02-26 20:40:45.330 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=logger, service=set_level>
2025-02-26 20:40:45.330 INFO (MainThread) [homeassistant.setup] Setup of domain logger took 0.0 seconds
2025-02-26 20:40:45.330 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event component_loaded[L]: component=logger>
2025-02-26 20:40:45.330 INFO (MainThread) [homeassistant.bootstrap] Setting up frontend: {'frontend'}
2025-02-26 20:40:45.331 DEBUG (MainThread) [homeassistant.setup] Dependency frontend will wait for dependencies ['api', 'auth', 'config', 'device_automation', 'diagnostics', 'file_upload', 'http', 'lovelace', 'onboarding', 'repairs', 'search', 'websocket_api']
2025-02-26 20:40:45.331 DEBUG (MainThread) [homeassistant.setup] Dependency api will wait for dependencies ['http']
2025-02-26 20:40:45.331 DEBUG (MainThread) [homeassistant.setup] Dependency auth will wait for dependencies ['http']
2025-02-26 20:40:45.331 DEBUG (MainThread) [homeassistant.setup] Dependency config will wait for dependencies ['http']
2025-02-26 20:40:45.343 INFO (MainThread) [homeassistant.setup] Setting up device_automation
2025-02-26 20:40:45.343 INFO (MainThread) [homeassistant.setup] Setup of domain device_automation took 0.0 seconds
2025-02-26 20:40:45.343 DEBUG (MainThread) [homeassistant.setup] Dependency diagnostics will wait for dependencies ['http']
2025-02-26 20:40:45.343 DEBUG (MainThread) [homeassistant.setup] Dependency file_upload will wait for dependencies ['http']
2025-02-26 20:40:45.406 INFO (MainThread) [homeassistant.setup] Setting up lovelace
2025-02-26 20:40:45.406 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event panels_updated[L]>
2025-02-26 20:40:45.407 DEBUG (MainThread) [homeassistant.setup] Dependency onboarding will wait for dependencies ['analytics', 'auth', 'http', 'person']
2025-02-26 20:40:45.407 DEBUG (MainThread) [homeassistant.setup] Dependency repairs will wait for dependencies ['http']
2025-02-26 20:40:45.407 DEBUG (MainThread) [homeassistant.setup] Dependency search will wait for dependencies ['websocket_api']
2025-02-26 20:40:45.407 DEBUG (MainThread) [homeassistant.setup] Dependency websocket_api will wait for dependencies ['http']
2025-02-26 20:40:45.407 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event component_loaded[L]: component=device_automation>
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.config] Invalid config for 'http' at configuration.yaml, line 5: some but not all values in the same group of inclusion 'proxy' 'http-><proxy>', got None, please check the docs at https://www.home-assistant.io/integrations/http
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Setup failed for 'http': Invalid config.
2025-02-26 20:40:45.408 DEBUG (MainThread) [homeassistant.setup] Dependency analytics will wait for dependencies ['api', 'websocket_api']
2025-02-26 20:40:45.408 DEBUG (MainThread) [homeassistant.setup] Dependency person will wait for dependencies ['image_upload', 'http']
2025-02-26 20:40:45.408 DEBUG (MainThread) [homeassistant.setup] Dependency image_upload will wait for dependencies ['http']
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'api'. Setup failed for dependencies: http
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Setup failed for 'api': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'auth'. Setup failed for dependencies: http
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Setup failed for 'auth': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'config'. Setup failed for dependencies: http
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Setup failed for 'config': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'diagnostics'. Setup failed for dependencies: http
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Setup failed for 'diagnostics': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'file_upload'. Setup failed for dependencies: http
2025-02-26 20:40:45.408 ERROR (MainThread) [homeassistant.setup] Setup failed for 'file_upload': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'repairs'. Setup failed for dependencies: http
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Setup failed for 'repairs': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'websocket_api'. Setup failed for dependencies: http
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Setup failed for 'websocket_api': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'image_upload'. Setup failed for dependencies: http
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Setup failed for 'image_upload': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'search'. Setup failed for dependencies: websocket_api
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Setup failed for 'search': (DependencyError(...), 'Could not setup dependencies: websocket_api')
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'analytics'. Setup failed for dependencies: api, websocket_api
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Setup failed for 'analytics': (DependencyError(...), 'Could not setup dependencies: api, websocket_api')
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'person'. Setup failed for dependencies: image_upload, http
2025-02-26 20:40:45.409 ERROR (MainThread) [homeassistant.setup] Setup failed for 'person': (DependencyError(...), 'Could not setup dependencies: image_upload, http')
2025-02-26 20:40:45.410 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'onboarding'. Setup failed for dependencies: analytics, auth, http, person
2025-02-26 20:40:45.410 ERROR (MainThread) [homeassistant.setup] Setup failed for 'onboarding': (DependencyError(...), 'Could not setup dependencies: analytics, auth, http, person')
2025-02-26 20:40:45.410 DEBUG (SyncWorker_1) [homeassistant.util.json] JSON file not found: /home/runner/workspace/config/.storage/lovelace_dashboards
2025-02-26 20:40:45.411 INFO (MainThread) [homeassistant.setup] Setup of domain lovelace took 0.0 seconds
2025-02-26 20:40:45.411 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event component_loaded[L]: component=lovelace>
2025-02-26 20:40:45.411 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'frontend'. Setup failed for dependencies: api, auth, config, diagnostics, file_upload, http, onboarding, repairs, search, websocket_api
2025-02-26 20:40:45.411 ERROR (MainThread) [homeassistant.setup] Setup failed for 'frontend': (DependencyError(...), 'Could not setup dependencies: api, auth, config, diagnostics, file_upload, http, onboarding, repairs, search, websocket_api')
2025-02-26 20:40:45.411 INFO (MainThread) [homeassistant.bootstrap] Setting up recorder: {'recorder'}
2025-02-26 20:40:45.738 INFO (MainThread) [homeassistant.setup] Setting up recorder
2025-02-26 20:40:45.738 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=recorder, service=purge>
2025-02-26 20:40:45.739 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=recorder, service=purge_entities>
2025-02-26 20:40:45.739 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=recorder, service=enable>
2025-02-26 20:40:45.739 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=recorder, service=disable>
2025-02-26 20:40:45.740 DEBUG (Recorder) [homeassistant.components.recorder.util] The last run ended at: 2025-02-26 20:37:17.316556+00:00 (now: 2025-02-26 20:40:45.740511+00:00)
2025-02-26 20:40:45.740 DEBUG (Recorder) [homeassistant.components.recorder.util] The system was restarted cleanly and passed the basic sanity check
2025-02-26 20:40:45.742 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Created new connection <sqlite3.Connection object at 0x7f4f3b7164d0>
2025-02-26 20:40:45.743 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> checked out from pool
2025-02-26 20:40:45.745 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> being returned to pool
2025-02-26 20:40:45.745 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> rollback-on-return
2025-02-26 20:40:45.745 DEBUG (Recorder) [homeassistant.components.recorder.core] Connected to recorder database
2025-02-26 20:40:45.745 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> checked out from pool
2025-02-26 20:40:45.748 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> being returned to pool
2025-02-26 20:40:45.748 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> rollback-on-return
2025-02-26 20:40:45.748 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> checked out from pool
2025-02-26 20:40:45.748 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> being returned to pool
2025-02-26 20:40:45.748 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> rollback-on-return
2025-02-26 20:40:45.749 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> checked out from pool
2025-02-26 20:40:45.805 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> being returned to pool
2025-02-26 20:40:45.805 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> rollback-on-return
2025-02-26 20:40:45.807 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> checked out from pool
2025-02-26 20:40:45.813 DEBUG (Recorder) [homeassistant.components.recorder.core] Activating event_types manager as all data is migrated
2025-02-26 20:40:45.814 DEBUG (Recorder) [homeassistant.components.recorder.core] Activating states_meta manager as all data is migrated
2025-02-26 20:40:45.815 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> being returned to pool
2025-02-26 20:40:45.815 DEBUG (Recorder) [homeassistant.components.recorder.pool.RecorderPool] Connection <sqlite3.Connection object at 0x7f4f3b7164d0> rollback-on-return
2025-02-26 20:40:45.816 INFO (MainThread) [homeassistant.setup] Setup of domain recorder took 0.1 seconds
2025-02-26 20:40:45.816 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event component_loaded[L]: component=recorder>
2025-02-26 20:40:45.816 INFO (MainThread) [homeassistant.bootstrap] Setting up stage 1: {'webhook', 'api', 'network', 'websocket_api', 'usb', 'zeroconf', 'cloud', 'bluetooth', 'dhcp', 'http', 'ssdp'}
2025-02-26 20:40:45.817 DEBUG (MainThread) [homeassistant.setup] Dependency webhook will wait for dependencies ['http']
2025-02-26 20:40:45.817 DEBUG (MainThread) [homeassistant.setup] Dependency network will wait for dependencies ['websocket_api']
2025-02-26 20:40:45.817 DEBUG (MainThread) [homeassistant.setup] Dependency usb will wait for dependencies ['websocket_api']
2025-02-26 20:40:45.817 DEBUG (MainThread) [homeassistant.setup] Dependency zeroconf will wait for dependencies ['network', 'api']
2025-02-26 20:40:45.817 DEBUG (MainThread) [homeassistant.setup] Dependency cloud will wait for dependencies ['http', 'webhook']
2025-02-26 20:40:45.817 DEBUG (MainThread) [homeassistant.setup] Dependency bluetooth will wait for dependencies ['usb']
2025-02-26 20:40:45.817 DEBUG (MainThread) [homeassistant.setup] Dependency ssdp will wait for dependencies ['network']
2025-02-26 20:40:45.817 DEBUG (MainThread) [homeassistant.setup] Dependency ssdp will wait for after dependencies ['zeroconf']
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'webhook'. Setup failed for dependencies: http
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Setup failed for 'webhook': (DependencyError(...), 'Could not setup dependencies: http')
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'network'. Setup failed for dependencies: websocket_api
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Setup failed for 'network': (DependencyError(...), 'Could not setup dependencies: websocket_api')
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'usb'. Setup failed for dependencies: websocket_api
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Setup failed for 'usb': (DependencyError(...), 'Could not setup dependencies: websocket_api')
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'cloud'. Setup failed for dependencies: http, webhook
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Setup failed for 'cloud': (DependencyError(...), 'Could not setup dependencies: http, webhook')
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'zeroconf'. Setup failed for dependencies: network, api
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Setup failed for 'zeroconf': (DependencyError(...), 'Could not setup dependencies: network, api')
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'bluetooth'. Setup failed for dependencies: usb
2025-02-26 20:40:45.818 ERROR (MainThread) [homeassistant.setup] Setup failed for 'bluetooth': (DependencyError(...), 'Could not setup dependencies: usb')
2025-02-26 20:40:45.819 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of 'ssdp'. Setup failed for dependencies: network
2025-02-26 20:40:45.819 ERROR (MainThread) [homeassistant.setup] Setup failed for 'ssdp': (DependencyError(...), 'Could not setup dependencies: network')
2025-02-26 20:40:45.822 INFO (SyncWorker_3) [homeassistant.util.package] Attempting install of scapy==2.5.0
2025-02-26 20:40:45.822 DEBUG (SyncWorker_3) [homeassistant.util.package] Running pip command: args=['/home/runner/workspace/.pythonlibs/bin/python', '-m', 'pip', 'install', '--quiet', 'scapy==2.5.0', '--timeout', '60', '--upgrade', '--constraint', '/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/homeassistant/package_constraints.txt']
2025-02-26 20:40:46.309 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:46.310 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
2025-02-26 20:40:47.318 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:47.319 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
2025-02-26 20:40:48.319 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:48.319 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
2025-02-26 20:40:49.320 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:49.320 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
2025-02-26 20:40:50.321 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:50.321 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
2025-02-26 20:40:51.322 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:51.322 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
2025-02-26 20:40:52.323 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:52.323 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
2025-02-26 20:40:52.776 INFO (SyncWorker_3) [homeassistant.util.package] Attempting install of aiodiscover==1.5.1
2025-02-26 20:40:52.776 DEBUG (SyncWorker_3) [homeassistant.util.package] Running pip command: args=['/home/runner/workspace/.pythonlibs/bin/python', '-m', 'pip', 'install', '--quiet', 'aiodiscover==1.5.1', '--timeout', '60', '--upgrade', '--constraint', '/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/homeassistant/package_constraints.txt']
2025-02-26 20:40:53.324 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:53.324 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
2025-02-26 20:40:54.325 DEBUG (MainThread) [homeassistant.bootstrap] Running timeout Zones: {'dhcp': <dhcp: 0 / 1>}
2025-02-26 20:40:54.325 DEBUG (MainThread) [homeassistant.bootstrap] Integration remaining: {}
